arches_lingo/src/arches_lingo/components/scheme/report/SchemeStandard.vue

```diff
@@ -52,34 +61,84 @@ async function getOptions(): Promise<ResourceInstanceReference[]> {
 }
 
 async function save() {
-    await updateSchemeCreation(
-        route.params.id as string,
-        schemeInstance.value as SchemeInstance,
-    );
+    try {
+        await updateSchemeCreation(
+            route.params.id as string,
+            schemeInstance.value as SchemeInstance,
+        );
+    } catch (error) {
+        toast.add({
+            severity: ERROR,
+            summary: $gettext("Error"),
+            detail:
+                error instanceof Error
+                    ? error.message
+                    : $gettext("Could not save the scheme standard"),
+        });
+    }
 
     getSectionValue();
+    emit(UPDATED);
```

Comment:
nit: would move these inside the `try` as these will still run even if that errors